### PR TITLE
[12.0][MIG] web_widget_url_advanced

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

File: web_widget_url_advanced/static/src/js/url_widget.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    basic_fields.UrlWidget.include({
+        /**
+         * @override
+         */
+        init: function () {
+            this._super.apply(this, arguments);
+            // retrieve customized `<a />` text from a field
+            // via `text_field` attribute or `options.text_field`
+            // NOTE: `<field text_field="foo"..` is not supported in tree views.
```
Comment (Tardo) [inline web_widget_url_advanced/static/src/js/url_widget.js]: Please, add this in ROADMAP.

**Review by Tardo — CHANGES_REQUESTED** [review]

File: web_widget_url_advanced/static/src/js/url_widget.js (hunk 1 above)
Comment (alexey-pelykh) [reply web_widget_url_advanced/static/src/js/url_widget.js]: Thanks, done!

**Review by simahawk — APPROVED** [review]

**Comment by Swapnesh-SerpentCS** [comment]
We can add https://github.com/OCA/web/pull/1234#issuecomment-511708931 here too so we don't need an additional module.

**Comment by simahawk** [comment]
@Swapnesh-SerpentCS can be done in second step after this PR is merged. Please review it to speed it up ;)
@Tardo good for you?

**Review by Tardo — CHANGES_REQUESTED** [review]
I see that when you get the "text_field" empty... the field with the url widget doesn't update the text until the page is refreshed.

url_widget

Please, fix or add it to roadmap.

**Comment by alexey-pelykh** [comment]
@simahawk should that be changed? I think it's due to `if (field_value)` check?

**Comment by alexey-pelykh** [comment]
@Tardo I've adjusted the implementation to update in any case as long as field is set

**Review by Tardo — APPROVED** [review]
LGTM :+1:, great!

**Comment by simahawk** [comment]
@alexey-pelykh @Tardo sorry guys, I missed it :stuck_out_tongue: You managed it so: let's merge it :)

**Pull request closed without merging**